GcExtensions.md

```diff
@@ -0,0 +1,429 @@
+# Garbage Collector Extensions
+
+- **Title:** Garbage collector extensions for better foreign language support
+- **Author:** Reimer Behrends <behrends@gmail.com>
+- **Created:** May 2018
+- **Status:** work in progress
```

Comment:
Can change this to `Accepted`, or we can wait for the corresponding PR (in case there are any changes)